### PR TITLE
Fix some english spelling errors

### PR DESCRIPTION
**Pull request by Jonasdero**

### CODE REVIEW

File: TechnicalWhitePaper_ENG.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ In the usage rental model, a service provider on top of MediBloc platform can st
 
 The fundamental principle of the usage rental model can be applied not only to the bandwidth model but also to the fee-based model. In this case, the term "pay-on-behalf" rather than "rental" would be the better term for the usage rights.
 
-The pay-on-behalf payment model can have both advantages and disadvantages compared to the usage rental model. MediBloc platform adopted the usage rental model based on the bandwidth model because in the fee-based model, the number of cryptocurrency of secondary signer decreases as the user uses the service, whereas the rental model offers the stability of the service and convenience for users.
+The pay-on-behalf payment model can have both advantages and disadvantages compared to the usage rental model. MediBloc platform adopted the usage rental model based on the bandwidth model because in the fee-based model, the amount of a certain cryptocurrency a secondary signer gets in return, decreases as the user uses the service, whereas the rental model offers the stability of the service and convenience for users.
```
Comment (bauser) [inline TechnicalWhitePaper_ENG.md]: the amount of cryptocurrency, in other words balance of MED, decreases as the user uses the service

Comment (bauser) [reply TechnicalWhitePaper_ENG.md]: above is what I exactly meant in Korean version

**Comment by Jonasdero** [comment]
Reverted the version number and integrated your comment 👍

**Comment by bauser** [comment]
Thank you for your works. 👍 Have a good weekend! :)

**Pull request merged**